ansible/tasks/internal/optimizations.yml

```diff
@@ -0,0 +1,15 @@
+- name: ensure services are stopped
+  community.general.snap:
+    name: amazon-ssm-agent
+    state: absent
+
+- name: ensure services are stopped and disabled for first boot
+  systemd:
+    enabled: no
+    name: '{{ item }}'
+    state: stopped
+  with_items:
+    - snapd
+    - postgresql
```

Comment:
for `postgresql`, correct me if I'm mistaken but I believe we would need it to have it started since we're running the `.sql` scripts at boot (then restarting it)

Reply:
Ah right, yeah

Reply:
Going to leave it stopped per offline discussion; we can start it up once it's been configured, before the init scripts need to be run.